lib/kubernetes-deploy/runner.rb

```diff
@@ -405,8 +405,12 @@ def confirm_context_exists
     end
 
     def confirm_namespace_exists
-      _, _, st = kubectl.run("get", "namespace", @namespace, use_namespace: false, log_failure: false)
-      raise FatalDeploymentError, "Namespace #{@namespace} not found" unless st.success?
+      st, err = nil
+      3.times do
```

Comment:
Should we have a small backoff period? Doesn't have to be exponential but I would not retry in a tight loop.

Reply:
I can add a small backoff 👍